lib/vfscore/vnode.c

```diff
@@ -450,7 +450,11 @@ vnode_dump(void)
 	struct vnode *vp;
 	struct mount *mp;
 	char type[][6] = { "VNON ", "VREG ", "VDIR ", "VBLK ", "VCHR ",
-			   "VLNK ", "VSOCK", "VFIFO" };
+			   "VLNK ", "VSOCK", "VFIFO",
+#ifdef CONFIG_LIBPOSIX_EVENT
+			   "VEPOLL", "VEVENT",
+#endif /* CONFIG_LIBPOSIX_EVENT */
+			 };
```

Comment:
Oh, yeah! Good catch. The fun fact is that I can remember counting the number of chars to make sure but forgot the `\0` 🤦🏻‍♂️ I changed the strings to `VEPLL` and `VEVNT` to avoid breaking the formatting of the debug output.